core/options.ts

```diff
@@ -307,77 +252,43 @@ export class Options {
    * @param options Dictionary of options.
    * @return Normalized zoom options.
    */
-  private static parseZoomOptions_(options: AnyDuringMigration): ZoomOptions {
+  private static parseZoomOptions_(options: BlocklyOptions): ZoomOptions {
     const zoom = options['zoom'] || {};
     const zoomOptions = {} as ZoomOptions;
     if (zoom['controls'] === undefined) {
-      // AnyDuringMigration because:  Property 'controls' does not exist on type
-      // '{}'.
-      (zoomOptions as AnyDuringMigration).controls = false;
+      zoomOptions.controls = false;
```

Comment:
This says it used to be AnyDuringMigration because the property doesn't exist on type {}, but the declaration + initialization of `zoomOptions` didn't change. What changed to make the AnyDuringMigration unncessary?

Same question for zoom, grid, and move options.

Reply:
AnyDuringMigration was introduced by the conversion script, but subsequent to that we've already done some work cleaning up types and such to get things to build. I'm guessing either the cast to ZoomOptions or the actual fields/types of ZoomOptions were fixed after the initial migration and provided the type information that was lacking at the time of conversion. It's also possible the migration script may have just been unsure of the type for whatever reason and erred on the side of caution.